.github/workflows/ci-cmake-options.yml

```diff
@@ -62,6 +62,15 @@ jobs:
         export VERBOSE=1
         make install -j$(nproc)
 
+    - name: CMake Include systemd unit
+      run: |
+        cmake --version
+        rm -rf ${{ github.workspace }}/build
+        cmake -B ${{ github.workspace }}/build -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/build -DLINUX_SYSTEMD=TRUE -DLINUX_SYSTEMD_UNITDIR=${{ github.workspace }}/build
+        cd ${{ github.workspace }}/build
+        export VERBOSE=1
+        make install -j$(nproc)
+
```

Comment:
this workflow does not run on this repo. Could you please extend .github/workflows/linux_make.yml as well? .github/workflows/linux_make.yml will run here

Reply:
Can do, does the latest push look better?

Reply:
adding a new "build-systemd" job would be cleaner

Reply:
Done.